modules/otel-forwarding-traces.adoc

```diff
@@ -39,12 +39,18 @@ metadata:
 rules:
   # <1>
   # <2>
-- apiGroups: ["", "config.openshift.io"]
-  resources: ["pods", "namespaces", "infrastructures", "infrastructures/status"]
+- apiGroups: [""]
+  resources: ["pods", namespaces",]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["apps"]
+  resources: ["replicasets"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["config.openshift.io"]
+  resources: ["infrastructures", "infrastructures/status"]
   verbs: ["get", "watch", "list"]
 ----
-<1> The `k8sattributesprocessor` requires permissions for pods and namespaces resources.
-<2> The `resourcedetectionprocessor` requires permissions for infrastructures and status.
+<1> Note that the Kubernetes Attributes Processor requires adding the permissions for the `pods`, `namespaces`, and `replicasets` resources.
+<2> Note that the Resource Detection Processor requires adding the permissions for the `infrastructures` and `status` resources.
```

Comment:
@pavolloffay, the permissions in these two callouts are already taken care of by adding the three `verbs` lines or must the user complete some additional steps? In other words, are these callouts saying that the `verbs` lines take care of this, or are these callouts saying that the user has to do some other steps elsewhere in addition to these `verbs` lines?

Reply:
Thank for the reply over Slack, Pavol.